src/js/components/Sidebar/index.d.ts

```diff
@@ -6,8 +6,10 @@ export interface SidebarProps {
   header?: React.ReactNode;
 }
 
-declare const Sidebar: React.FC<BoxProps &
+export type SidebarExtendedProps = BoxProps &
```

Comment:
Use this as references to solve properties conflict: https://github.com/grommet/grommet/pull/5021/files#diff-37524e012a5eb198c557ffc83a19486994866790935ef45e467d68096560ca90R26